src/components/utils/src/file_system.cc

```diff
@@ -100,9 +94,6 @@ size_t file_system::DirectorySize(const std::string& path) {
     }
   }
   closedir(directory);
-#ifdef __QNXNTO__
```

Comment:
@dev-gh Right but these changes are critical and needed for further development.